modin/core/dataframe/pandas/dataframe/dataframe.py

```diff
@@ -2171,12 +2171,44 @@ def map(
         PandasDataframe
             A new dataframe.
         """
-        map_fn = (
-            self._partition_mgr_cls.lazy_map_partitions
-            if lazy
-            else self._partition_mgr_cls.map_partitions
-        )
-        new_partitions = map_fn(self._partitions, func, func_args, func_kwargs)
+        if self.num_parts <= 1.5 * CpuCount.get():
```

Comment:
Why do we have 1.5?,  should this be configuable?

Reply:
It is a good point, I wanted to discuss it.
This magic number is also used in `rebalance_partitions` function for the same reason as here. As I can see, we don't expect that this parameter may be changed by user, so I would like to suggest using a general constant for this. 
@YarShev What do you think about it?

Reply:
It is an implementation-details. I don't think we should expose this kind of parameter, at least, for now.